src/header/HeaderBAM.js

```diff
@@ -44,7 +44,7 @@ const HeaderBAM = ({ className }) => {
                     }`}>
                     <span>
                         BAM &nbsp;
-                        <SvgIcon id='link-icon' icon={faLink} />
+                        <SvgIcon id='link-icon' icon={Link} />
```

Comment:
I see that `<SvgIcon>` component from pelagos has been deprecated
`/** @deprecated use a Carbon icon instead. */
`
could you try using carbon icon instead directly.

Example like this  ->  https://carbondesignsystem.com/elements/icons/code#usage

Reply:
Of course, thank you for your advice.